### ISSUE TITLE
Add --diff to black command

### ISSUE THREAD

**Issue by tannewt** [adafruit/cookiecutter-adafruit-circuitpython#78]
That way we can hand-fix small PRs that break formatting.

**Comment by dherrada** [comment]
@Flameeyes Do you know how we can do this now that we're using pre-commit?

**Comment by Flameeyes** [comment]
Good question — the [hook documentation](https://black.readthedocs.io/en/stable/version_control_integration.html) says not to use `args`, but that implies that `args` exists… so it would just be a matter of adding that to the hook.

But of course that would print the diff and fail on pre-commit instead of fixing it when running pre-commit locally.

**Comment by dherrada** [comment]
@tannewt Is this still something you'd like me to try to do given that it could cause issues when running locally

**Comment by tannewt** [comment]
Can we run it twice or call regular diff afterwards?

**Comment by dherrada** [comment]
@tannewt Yeah, I think that should work. I can try to take a look at doing this sometime soon.
